test/util_test.sh

```diff
@@ -0,0 +1,300 @@
+#!/bin/bash
+# This file ensures that the helper functions in util.sh behave as expected
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/util.sh"
+source "${OS_ROOT}/test/util.sh"
+os::log::install_errexit
+
+function echo_and_exit() {
+	echo $1
+	exit $2
+}
+
+mkdir -p /tmp/openshift/origin/test/cmd/
+JUNIT_OUTPUT_FILE=/tmp/openshift-cmd/junit_output.txt
+
+VERBOSE=1
+
+# positive tests
```

Comment:
I want an example that exercises embedded commands (`echo "this $(echo is) real"`)